src/api/ping.ts

```diff
@@ -0,0 +1,9 @@
+import { Client } from '.'
+
+export const ping = (client: Client) => async () => {
+  return await fetch(`${client.baseUrl}/ping`, {
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  }).then((res) => res.text())
```

Comment:
async function つかうなら
then 使わないほうが手続き的でいいかもしれん

(ほかのとこも)

```suggestion
  const res = await fetch(`${client.baseUrl}/ping`, {
    headers: {
      'Content-Type': 'application/json',
    },
  })
  return res.text()
```